### ISSUE TITLE
win hosts are not starting the swarming process on reboots

### ISSUE THREAD

**Issue by yusuf-goog** [flutter/flutter#94761]
when some win host restart (e.g. win-4) it doesn't restart the swarming bot.

**Comment by yusuf-goog** [comment]
So in some new releases of windows, the checkbox to not require username/password for sign-in on restart does not show up. This means that the swarming user does not auto sign-in on restart, and subsequently does not start the swarming bot process.

To get back the checkbox, see: https://www.askvg.com/fix-users-must-enter-a-user-name-and-password-to-use-this-computer-checkbox-missing-in-windows-10/

Basically, we need to run this, which adds the necessary registry entry to get the checkbox to show up:

reg ADD “HKLM\SOFTWARE\Microsoft\Windows NT\CurrentVersion\PasswordLess\Device” /v DevicePasswordLessBuildVersion /t REG_DWORD /d 0 /f

then you can run `netplwiz` and uncheck the checkbox for the swarming user.
